### DISCUSSION TITLE
v3.6.1

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#347] [Announcements]
### Bug Fixes

- New 403 errors (YouTube)

<hr/>
If you're enjoying PipePipe, please help share it with your friends. Every bit of support means a lot to us!
<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.6.1'>v3.6.1</a>.</em>

**Comment by Asukitta** [comment]
It didnt let me install the x86 and the universal versión: It gives me this error esch time I've tried

[Screenshot_20240927-062031.png](https://github.com/user-attachments/assets/68f2e9d7-d636-42b6-960c-683a4def7c5

**Reply by InfinityLoop1308** [reply]
I can't see your attachment.

**Reply by InfinityLoop1308** [reply]
And, the current version is 3.8.2. You can move to that thread.

**Reply by InfinityLoop1308** [reply]
https://github.com/InfinityLoop1308/PipePipe/discussions/452
